WordPress/Classes/Utility/HelpshiftUtils.m

```diff
@@ -15,7 +14,7 @@
 NSString *const UserDefaultsHelpshiftWasUsed = @"wp_helpshift_used";
 NSString *const HelpshiftUnreadCountUpdatedNotification = @"HelpshiftUnreadCountUpdatedNotification";
 // This delay is required to give some time to Mixpanel to update the remote variable
-CGFloat const HelpshiftFlagCheckDelay = 10.0;
+CGFloat const HelpshiftFlagCheckDelay = 2.0;
```

Comment:
Only because I didn't want to remove the mechanism but also didn't want it to happen too quickly in case I'm breaking something. I'm going to do another PR soon to replace this mechanism. ;)